modules/oauthlogin/oauth.go

```diff
@@ -169,15 +197,22 @@ func CallbackOAuth(tempCode, state, serviceType string) (OAuthResponse, error) {
 		return OAuthResponse{}, err
 	}
 	defer res.Body.Close()
+
+	resBody, err := io.ReadAll(res.Body)
```

Comment:
What would you suggest doing instead?

Reply:
What we previously did, stream it into the JSON decoder.

Reply:
It shouldn't matter as these are small payloads, right? I could remove it here, but below I need to save the raw data (which is likely to get removed at some point, but I want to prevent yet another migration ahead of time). Besides, this "glue" code will be going away the moment we add one or two more OAuth providers.

Reply:
Well let's at least add [io.LimitedReader](https://pkg.go.dev/io#LimitedReader) just to be safe.

Reply:
Feel free to add it.